examples/plot_vector_testing.py

```diff
@@ -0,0 +1,205 @@
+"""
+Testing Vector Plots Created Using Geopandas with MatPlotCheck
+==============================================================
+
+This vignette will show you how to use Matplotcheck to test spatial vector data
+plots created using Geopandas.
+
+"""
+
+################################################################################
+# Setup
+# -----
+# To begin, import the libraries that you need.
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from shapely.geometry import Polygon, LineString
+import geopandas as gpd
+from matplotcheck.vector import VectorTester
+
+################################################################################
+# Create Geometry Objects
+# -----------------------
+# To run this test, you first must plot some example data. Below are GeoPandas
+# dataframes created to replicate data that may be used in an earth data
+# science exercise. The polygons could be study areas, the lines could be roads
+# and streams near those study areas, and the points could be measurments
+# within the study areas.
+
+# Create a polygon GeoDataFrame
+coords = [(2, 4), (2, 4.25), (4.25, 4.25), (4.25, 2), (2, 2)]
+coords_b = [(i[0]+5, i[1]+7) for i in coords]
+polygon_gdf = gpd.GeoDataFrame(
+    [1, 2], geometry=[Polygon(coords), Polygon(coords_b)], crs="epsg:4326")
+polygon_gdf["attr"] = ["Area 1", "Area 2"]
+
+# Create a line GeoDataFrame
+linea = LineString([(1, 1), (2, 2), (3, 2), (5, 3)])
+lineb = LineString([(3, 4), (5, 7), (12, 2), (10, 5), (9, 7.5)])
+line_gdf = gpd.GeoDataFrame([1, 2], geometry=[linea, lineb], crs="epsg:4326")
+
+# Create a multiline GeoDataFrame
+linec = LineString([(2, 1), (3, 1), (4, 1), (5, 2)])
+multi_line_gdf = gpd.GeoDataFrame(geometry=gpd.GeoSeries(
+    [line_gdf.unary_union, linec]), crs="epsg:4326")
+multi_line_gdf["attr"] = ["road", "stream"]
+
+# Create a point GeoDataFrame
+
+points = pd.DataFrame(
+    {
+        "lat": np.array([1, 2, 1, 0, 4]),
+        "lon": np.array([3, 4, 0, 0, 1]),
+    }
+)
+
+point_gdf = gpd.GeoDataFrame(
+    {"A": np.arange(5), "B": np.arange(5)},
+    geometry=gpd.points_from_xy(points.lon, points.lat), crs="epsg:4326"
+)
+point_gdf["size"] = [100, 100, 300, 300, 500]
+
+# Create symbology dictionary to use in the legend
+
+line_symb = {"road": "black", "stream": "blue"}
+point_symb = {100: "purple", 300: "green", 500: "brown"}
+
+################################################################################
+# Create Your Spatial Plot
+# ------------------------
+# Above you created several GeoPandas GeoDataFrame objects that you want
+# to plot. To plot these data according to attribute value, you can group
+# the geometry by attributes and plot within a loop. Once you've created
+# your plot, you are ready to test it using MatPlotCheck.
```

Comment:
Minor changes to remove contraction: "Once you have created your plot, you will be ready to test it using Matplotcheck"

Note change of capitalization of the package name to Matplotcheck.